oeps/oep-0021-proc-deprecation.rst

```diff
@@ -22,295 +22,323 @@ OEP-0021: Deprecation and Removal
-following for communications:
+Timeline
+~~~~~~~~
 
-1. Email the `edx-code list. <https://groups.google.com/forum/#!forum/edx-code>`_
-In the event that the edx-code list has been deprecated, its designated
-replacement channel should be utilized instead.
+When determining target dates to propose for the removal process, consider that
+it will vary depending on team resources and the technology being removed.
+The importance of removal (as described in Motivation_) should be communicated
+with all team members so the removal can be prioritized and completed in a
+timely manner. A suggested timeline is shown in the diagram below, which
+considers the timing of the next `Open edX named release`_.
 
-2. Make an announcement in the **#architecture** channel in the Open edX Slack
-team. The announcement should be shared to **#general** as well. 
+.. image:: oep-0021/timeline.png
+   :align: center
+   :alt: A diagram that suggests having a 2 week time period between the 
+    *Proposed* and *Accepted* states, giving the community enough time to provide
+    feedback. After which, the *Deprecated*, *Removing*, and *Removed* transition
+    periods will vary by the type and scope of the technical change.
 
-3. If originating within edX, an additional message should be sent to the
-engineering-all email list. 
+* **Proposed** on Day 1
+* **Communicated** from Day 2 to Day 13
+* **Accepted** on Day 14 *(depending on influx of feedback)*
+* **Deprecated/Removing/Removed** - from Day 15 onwards *(depending on resources and technology being removed)*
 
-A message template with suggested topics to cover is below.
+.. _Open edX named release: http://open-edx-proposals.readthedocs.io/en/latest/oep-0010-proc-openedx-releases.html
 
-*Note: This template is oriented towards an email announcement to the
-edx-code mailing list but relevant portions can be repurposed for other
-channels such as the Open edX Slack team, internal edX mailing list and
-others.*
+Document
+~~~~~~~~
 
-   From: Deprecation originator
+Do the following to document your proposal (see DEPR-6_ as an example):
 
-   To: edx-code@googlegroups.com
+#. Create a **DEPR** ticket on the `Deprecation and Removal JIRA board`_.
+#. Include target dates for: *Accepted* and *Removed* (in explicit fields on the
+   ticket). See Timeline_ for considerations.
+#. Include in which `Open edX named release`_ the code will be removed.
+#. Include a description with links to what is being removed, in a
+   **Removal** section.
+#. Include a description with links to what it is being replaced by, in a
+   **Replacement** section.
+#. If you plan to mark the code for deprecation, explain how in a
+   **Deprecation** section. See Deprecated_ for considerations.
+#. If automated migration will be needed, explain your migration plan in a
+   **Migration** section.
+#. If there is any additional publicly shareable information or data from your
+   earlier analysis, include that on the ticket.
```

Comment:
Can you add a bullet to check that the ticket is open for all JIRA users?

Reply:
So.. when you ran into this access problem, the entire JIRA board for DEPR tickets were inaccessible to non-edX users.  Since then, we have updated the permissions.  Now it would actually be extra effort for someone to change the permissions on the ticket.  Do you think it's still worthwhile for us to add a bullet to have the ticket author double check the permissions?

Reply:
I double checked - there is not even a "Security Level" setting on these DEPR tickets - so one cannot change the access level of these tickets.